bindings/python/requirements-test.in

```diff
@@ -0,0 +1,7 @@
+iniconfig
```

Comment:
The main reason for us to keep frozen versions is to get a reproducible build between ci and local. If the default workflow is to generate updated versions it defeats this purpose. If the reproducibility is not a requirement this file could just define minimum versions?

Reply:
I see. I am torn here. On the one hand I like the idea of having a reproducible test case, but on the other hand I would also like to see if stuff breaks with newer versions. Any thoughts?